data/core/rootview.lua

```diff
@@ -372,7 +372,30 @@ end
 ---@return boolean
 function RootView:on_file_dropped(filename, x, y)
   local node = self.root_node:get_child_overlapping_point(x, y)
-  return node and node.active_view:on_file_dropped(filename, x, y)
+  local result = node and node.active_view:on_file_dropped(filename, x, y)
+  if not result then
```

Comment:
Nit: you could avoid indenting code by doing something like
```suggestion
  if result then return result end
```